locales/en.default.schema.json

```
@@ -1983,6 +1983,187 @@
       "presets": {
         "name": "Video"
       }
+    },
+    "slideshow": {
+      "name": "Slideshow",
+      "settings": {
+        "layout": {
+          "label": "Layout",
+          "options__1": {
+            "label": "Full width"
+          },
+          "options__2": {
+            "label": "Grid"
+          },
+          "options__3": {
+            "label": "Grid with preview"
+          }
+        },
+        "slide_height": {
+          "label": "Slide height",
+          "options__1": {
+            "label": "Adapt to first image"
+          },
+          "options__2": {
+            "label": "Small"
+          },
+          "options__3": {
+            "label": "Medium"
+          },
+          "options__4": {
+            "label": "Large"
+          }
+        },
+        "slider_visual": {
+          "label": "Pagination style",
+          "options__1": {
+            "label": "Fraction"
```

```suggestion
            "label": "Counter"
```
Ludo! We are reverting this version based on feedback. Sorry! cc. @katycobb 
This `1/2` style should be the default style when you add a section.